src/api/app/helpers/webui/project_helper.rb

```diff
@@ -4,10 +4,6 @@ module Webui::ProjectHelper
 
   protected
 
-  def escape_project_list(arr)
-    arr.map { |p| "['#{p[0]}','#{escape_javascript(p[1])}']" }.join(",\n").html_safe
```

Comment:
This was used on the project index page! Now we use the new helper from `` src/api/app/helpers/webui/webui_helper.rb ``